CHANGELOG.md

```diff
@@ -94,6 +94,7 @@ CHANGELOG
       new: `public function queryAndReturnResult($query, ?array $variables = [], array $opts = []): ExecutionResult`
 
 ### Added
+- The primary execution of the GraphQL requested is now modeled via middlewares [\#762 / crissi and mfn](https://github.com/rebing/graphql-laravel/pull/762)
```

Comment:
@crissi I don't want it to go unmentioned I also mentioned me as secondary author, is this fine with you?

Reply:
all good :slightly_smiling_face: